docs/build-guide.rst

```diff
@@ -27,35 +27,17 @@ add instructions for any other distributions.
 NixOS Linux
 ^^^^^^^^^^^
 
-First of all, install ``mono``, ``fsharp`` and ``dotnetPackages.Nuget``::
+Enter the development environment::
 
-    $ nix-env -i mono fsharp
-    $ nix-env -iA nixos.pkgs.dotnetPackages.Nuget
-
-Please note that you need ``fsharp`` >= 4.0 to build Naggum, which is currently
-in the unstable channel. So maybe you'll need to switch to the unstable channel
-or build ``fsharp`` from `Nixpkgs`_ source.
-
-According to the recommendations in `patch-fsharp-targets`_ build helper,
-you may want to set up the environment variable ``FSharpTargetsPath`` either
-globally or locally::
```

Comment:
Please delete the `Nixpkgs` and `patch-fsharp-targets` from the bottom of the file.